hummingbot/connector/exchange/kucoin/kucoin_exchange.pyx

```diff
@@ -1,1042 +0,0 @@
-import asyncio
```

Comment:
`kucoin_exchange.pxd` should be deleted too

Reply:
Good catch, thank you